### PR TITLE
Removed all hardcoded colorsss & fixed a few styling bugs

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #1188

I think I've captured them all... I think......

### CODE REVIEW

File: source/sass/global.scss [hunk 1]
```diff
@@ -9,14 +9,14 @@ body {
   .wrapper-2 {
     background:
       url(../_images/bg-white-lines.png) center top / auto 335px repeat,
-      linear-gradient(to left, rgba(61, 254, 253, 1) 0%, rgba(183, 212, 63, 1) 100%);
+      linear-gradient(to right, $neon-blue, $green);
```
Comment (mmmavis) [inline source/sass/global.scss]: verified with Kristina

File: source/sass/global.scss [hunk 2]
```diff
@@ -9,14 +9,14 @@ body {
...
     background-blend-mode: normal, normal;
   }
 
   // Homepage
   // &#view-home .wrapper-2 {
   //   background:
-  //     linear-gradient(to left, rgba(61, 254, 253, 1) 0%, rgba(183, 212, 63, 1) 100%),
+  //     linear-gradient(to right, $neon-blue, $green),
```
Comment (mmmavis) [inline source/sass/global.scss]: verified with Kristina

File: source/sass/global.scss [hunk 3]
```diff
@@ -28,7 +28,7 @@ body {
       background-blend-mode: hard-light;
       background-image:
         url(../_images/fellowships/hero-image@2x.png),
-        linear-gradient(to left, rgba(61, 254, 253, 1) 0%, rgba(183, 212, 63, 1) 100%);
+        linear-gradient(to right, $neon-blue, $green);
```
Comment (mmmavis) [inline source/sass/global.scss]: verified with Kristina

File: source/js/components/nav/nav.scss [hunk 4]
```diff
@@ -1,13 +1,13 @@
 #primary-nav header {
-  background: rgba(0, 0, 0, 0.9);
+  background: transparentize($black, 0.1);
```
Comment (gvn) [inline source/js/components/nav/nav.scss]: 🤣

File: source/js/components/nav/nav.scss [hunk 5]
```diff
@@ -50,10 +50,8 @@
 
       &:hover,
       &:focus {
-        color: rgba(255, 237, 0, 100);
+        color: $white;
```
Comment (gvn) [inline source/js/components/nav/nav.scss]: This should be `$yellow`, right?

File: source/js/components/nav/nav.scss [hunk 6]
```diff
@@ -50,10 +50,8 @@
...
         outline: 0;
-        text-decoration: none;
-        border-bottom: 5px solid $black;
```
Comment (gvn) [inline source/js/components/nav/nav.scss]: why is this being taken out?

File: source/js/components/nav/nav.scss [hunk 7]
```diff
@@ -68,7 +66,7 @@
     #view-about .about {
       color: $white;
       font-weight: 900;
-      border-bottom: 2px solid $white;
```
Comment (gvn) [inline source/js/components/nav/nav.scss]: why is this changed?

File: source/js/components/nav/nav.scss (hunk 5 above)
Comment (mmmavis) [reply source/js/components/nav/nav.scss]: Verified with Kristina on this. This actually fixes one of the issues mentioned in https://github.com/mozilla/foundation.mozilla.org/issues/1186

File: source/js/components/nav/nav.scss (hunk 6 above)
Comment (mmmavis) [reply source/js/components/nav/nav.scss]: changes approved by Kristina

File: source/js/components/nav/nav.scss (hunk 7 above)
Comment (mmmavis) [reply source/js/components/nav/nav.scss]: Since these elements are `<a>` themselves I thought using `text-decoration: underline` was a more common approach to get the underline shown?

Comment (mmmavis) [reply source/js/components/nav/nav.scss]: design has been approved by Kristina too

**Comment by mmmavis** [comment]
yassss 🍷
